core/bentley/src/Assert.ts

```diff
@@ -17,7 +19,55 @@
  * which indicate a misuse of the API which should be eliminated during development.
  * @public
  */
-export function assert(condition: boolean, msg?: string): asserts condition {
-  if (!condition)
-    throw new Error(`Assert: ${msg ?? "Programmer Error"}`);
+
+/** Asserts that a condition is `true` and - in development builds - throws an error if it is not.
```

Comment:
"in development builds" -> "when enabled"

Reply:
fixed.